src/components/container/SeverityCalendar/SeverityCalendar.tsx

```diff
@@ -145,6 +152,7 @@ export default function (props: SeverityCalendarProps) {
     if (!data) {
         return <LoadingIndicator innerRef={props.innerRef} />;
     } else {
-        return <Calendar innerRef={props.innerRef} className="mdhui-simple-calendar" year={intervalStart.getFullYear()} month={intervalStart.getMonth()} dayRenderer={renderDay} />;
+        return (!dateRangeContext || dateRangeContext?.intervalType === "Month") &&
```

Comment:
I don't think this condition is necessary.  You can just return the `Calendar` tag.